backend/src/main/resources/application.yaml

```diff
@@ -20,6 +20,12 @@ graphql:
     corsEnabled: true
   tools:
     schema-location-pattern: "**/*.graphqls"
+  datetime:
+    scalars:
+      LocalDate:
+        format: MM/dd/yyyy
```

Comment:
I feel like we just demonstrated adequately why we should not use this as a wireline format.